snitun/multiplexer/core.py

```diff
@@ -205,14 +217,15 @@ async def _runner(self) -> None:
 
     def _write_message(self, message: MultiplexerMessage) -> None:
         """Write message to peer."""
-        header = message.id.bytes
-        header += message.flow_type.to_bytes(1, byteorder="big")
-        header += len(message.data).to_bytes(4, byteorder="big")
```

Comment:
to_bytes unsigned is default https://docs.python.org/3/library/stdtypes.html#int.to_bytes